facebook-commerce-pixel-event.php

```diff
@@ -222,12 +228,11 @@ public static function set_basecode($new_basecode) {
 
   public static function get_basecode() {
     $fb_options = self::get_options();
-    $basecode = $fb_options[self::BASECODE_KEY];
-    if (!isset($basecode)) {
+    if (!isset($fb_options[self::BASECODE_KEY])) {
```

Comment:
We rewrote this function entirely and change it to just return the basecode. So you'll want to undo the changes here.